src/threads/table.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { dynamodb, toDBItem } from '../dynamodb';
+import { ForumTable } from './table';
+
+const putItem = (threadId: string, timestamp: number, data: Record<string, unknown>) => dynamodb.putItem({
+  TableName: 'forumTable',
+  Item: toDBItem({ threadId, timestamp, ...data }),
+});
+const getAll = (threadId: string) => dynamodb.query({
```

Comment:
For testing purposes, probably a getAll without `threadId` would be better... you have an empty db before the test, you add an element, then you can check that you have exactly one more element. Doing a get by id, you are checking what you expect to see.

Reply:
Fixed